<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudentDialog</class>
 <widget class="QDialog" name="AddStudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>583</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>13</y>
     <width>598</width>
     <height>560</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Coordonnées</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="Label_Matricule">
             <property name="text">
              <string>Matricule</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="LineEdit_Matricule"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="Label_Nom">
             <property name="text">
              <string>Nom</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="Label_Prenom">
             <property name="text">
              <string>Prénom</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="LineEdit_Nom"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="LineEdit_Prenom"/>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="Label_Adresse">
           <property name="text">
            <string>Adresse</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QTextEdit" name="TextEdit_Adresse"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Date de naissance</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QDateEdit" name="DateEdit_Naissance"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="Label_Telephone">
           <property name="text">
            <string>Télephone </string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLineEdit" name="LineEdit_Telephone"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="Label_Email">
           <property name="text">
            <string>E-mail</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLineEdit" name="LineEdit_Email"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Cours</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QListView" name="listView"/>
         </item>
        </layout>
        <zorder>listView</zorder>
        <zorder>groupBox</zorder>
        <zorder>groupBox</zorder>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Discard|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddStudentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddStudentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
